components/runtime/spidermonkey/patches/02-jsfunc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/js/src/jsfun.h b/js/src/jsfun.h
+index a52092c..9555a03 100644
+--- a/jsfun.h
++++ b/jsfun.h
+@@ -144,6 +144,7 @@ struct JSFunction : public JSObject_Slots2
+         void            *nativeOrScript;
+     } u;
+     JSAtom          *atom;        /* name for diagnostics and decompiling */
++    void            *dummy;       /* for building with Solaris Studio */
```

Comment:
seems to be relevant for studio only (based on comment). Is it true?

Reply:
I tested with the patch and all tests passed successfully.
So I would use it.